test/integration/patients/patient/patient-flow.js

```diff
@@ -432,139 +434,115 @@ context('patient flow page', function() {
     cy
       .routesForPatientAction()
       .routeFlow(fx => {
-        fx.data.id = '1';
-        fx.data.relationships['program-flow'] = { data: { id: '1' } };
-
-        fx.included.push({
-          id: '1',
-          type: 'program-actions',
-          attributes: {
-            name: 'Conditional',
-            published_at: testTs(),
-            archived_at: null,
-            behavior: 'conditional',
-            details: '',
-            days_until_due: 0,
-            sequence: 0,
-          },
+        fx.data = mergeJsonApi(testFlow, {
           relationships: {
-            owner: { data: null },
-            form: { data: { id: '11111' } },
+            'program-flow': getRelationship('1', 'program-flows'),
```

Comment:
I think we can use `getProgramFlow` here instead of an arbitrary id

Reply:
Updated 👍 had to do a little extra since that same program flow needs to be in the `fx.included`.